assets/boilerplate/parts/post-meta.html

```diff
@@ -0,0 +1,8 @@
+<!-- wp:group {"layout":{"type":"flex"}} -->
+<div class="wp-block-group">
+	<!-- wp:post-author {"showAvatar":false,"showBio":false,"style":{"typography":{"fontSize":"14px"}}} /-->
+	<!-- wp:post-date {"isLink":true,"style":{"typography":{"fontSize":"14px"}}} /-->
+	<!-- wp:post-terms {"term":"category","style":{"typography":{"fontSize":"14px"}}} /-->
+	<!-- wp:post-terms {"term": "post_tag","style":{"typography":{"fontSize":"14px"}}} /-->
+</div>
+<!-- /wp:group -->
```

Comment:
We can reduce this by:
- Setting font size in theme.json
- Or setting the font-size on the group.